test/testImageCommands.sh

```diff
@@ -1,82 +1,89 @@
 #!/usr/bin/env bash
 
-./ensureShunitIsPresent.sh
+# import functions that are shared across unit tests
+source PharoLauncherCommonFunctions.sh
 
-#setup pharo launcher and image name paths
-PHLSCRIPT=./pharo-launcher.sh
-IMAGE=./shared/PharoLauncher.image
+#ensure that Shell unit test library is installed
+ensureShunitIsPresent
 
-# setup commands for image manipulation
-processListCommand="runLauncherScript image processList"
-launchCommand="runLauncherScript image launch PhLTestImage"
-killAllCommand="runLauncherScript image kill --all"
-killCommand="runLauncherScript image kill PhLTestImage"
+#setup sample image name and template name
+SAMPLEIMAGE="PhLTestImage"
```

Comment:
I would use SAMPLE_IMAGE to improve readability

Reply:
Ok, will be renamed.